Code_Review_Assignment_JrSDET.java

```diff
@@ -1 +1,172 @@
+package com.org.happyfox;
+
+import java.awt.AWTException;
+import java.awt.Robot;
+import java.awt.event.KeyEvent;
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.interactions.Actions;
+
+public class Testcase101 {
+
+    public static void main(String[] args) throws InterruptedException, AWTException {
+        System.setProperty("webdriver.gecko.driver", "C:\\Users\\Johny\\Downloads\\geckodriver-v0.33.0-win64\\geckodriver.exe");
+        WebDriver driver = new FirefoxDriver();
+        driver.get("https://interview.supporthive.com/staff/");
+        driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
+        driver.manage().window().maximize();
+        driver.findElement(By.id("id_username")).sendKeys("Agent");

```

Comment:
id is wrong here. id is 'userName' here
<img width="1405" alt="Screenshot 2024-07-30 at 12 23 28 AM" src="https://github.com/user-attachments/assets/430d7431-ab01-4671-a06c-ada05572ae0f">